### DISCUSSION TITLE
Rename project BaGet to BaGetter

### DISCUSSION THREAD

**Discussion by seriouz** [bagetter/BaGetter#5] [General]
Please rename the BaGet project to BaGetter.
The maintainer of BaGet whished that a possible fork should be named other then BaGet: https://github.com/loic-sharma/BaGet/discussions/780#discussioncomment-6742707

Thank you!

**Comment by bagetter-admin** [comment]
I've done a first pass at renaming things, still need to go through each file but most of it should be done.
I've also done an update to .NET 6.0 and updated dependencies too. The Azure Blob Storage provider will need work though as the old code was using an API that is now deprecated.
Builds are working ok but the Release needs more work.

**Comment by daghsentinel** [comment]
A few questions for the new fork:

1. How can we help?
2. A working docker image on a public repo would be a great start, but that implies building, uploading and making it available. Any plan on doing this?

**Reply by robford123** [reply]
Sorry there's not been much progress yet, hopefully I will have time soon to dedicate. There is a pre-release docker image available running in .NET6 for you to try out:

docker pull bagetter/bagetter:latest

Let me know if you want to get involved in any way, shape or form.

**Reply by daghsentinel** [reply]
I am a .NET backender as long as it has existed, and an avid user of nuget.org, myget.org etc. I have also contributed to a lot of free projects, and would like to contribute in any way I can. If you have any idea on what the most pressing issues are, I can surely join in on them.

**Reply by seriouz** [reply]
A dotnet 7 or even 8 (gets released next month) version would be good.

**Reply by daghsentinel** [reply]
It's in a pull request already ;)

**Reply by seriouz** [reply]
Seems good 👍.
Another idea: You could look at the PRs or issues of BaGet and pick important looking things and apply them onto BaGetter ;D

**Comment by seriouz** [comment]
Everything solved! 💪
